itou/invitations/models.py

```diff
@@ -33,13 +40,18 @@ def pending(self):
 
 
 class InvitationAbstract(models.Model):
-    EXPIRATION_DAYS = 14
+    DEFAULT_VALIDITY_DAYS = 14
 
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     email = models.EmailField(verbose_name="e-mail")
     first_name = models.CharField(verbose_name="prénom", max_length=255)
     last_name = models.CharField(verbose_name="nom", max_length=255)
     sent = models.BooleanField(verbose_name="envoyée", default=False)
+    validity_days = models.PositiveSmallIntegerField(
+        verbose_name="durée de validité en jours",
+        default=DEFAULT_VALIDITY_DAYS,
+        validators=[MinValueValidator(DEFAULT_VALIDITY_DAYS), MaxValueValidator(90)],
```

Comment:
Tant qu’à faire, on pourrait imaginer avoir des invitations valides pour une seule journée. Ça permettrait de faire des invitations de durée plus courte à l’avenir.
```suggestion
        validators=[MinValueValidator(1), MaxValueValidator(90)],
```